scripts/obs_im_calc.sl

```diff
@@ -23,7 +23,7 @@ do
     if [[  `find ${D} -name accBB | wc -l` -ge 1 ]]
     then
         fault_name=`basename $D`
-        python $gmsim/workflow/scripts/im_calc_checkpoint.py ${obs_dirs}/IM_calc/ $((`ls $D/*/*/accBB | wc -l` / 3)) --event_name ${fault_name} --observed
+        python $gmsim/workflow/scripts/im_calc_checkpoint.py ${obs_dirs}/IM_calc/ $((`ls $D/*/*/accBB | wc -l` / 3)) 1 --event_name ${fault_name} --observed
```

Comment:
to make it easier for future usage without looking into the checkpoint script what '1' stands for,
i suggest making the comp a variable.
for example
```suggestion
       comp="000 090 geom"
       comp_count=`echo $comp | awk '{print NF}' `
       python $gmsim/workflow/scripts/im_calc_checkpoint.py ${obs_dirs}/IM_calc/ $((`ls $D/*/*/accBB | wc -l` / 3)) $comp_count --event_name ${fault_name} --observed
```
and 
```suggestion
        time python ${IMPATH}/calculate_ims.py $D/*/*/accBB a -o ${obs_dirs}/IM_calc/ -np ${SLURM_CPUS_PER_TASK} -i ${fault_name} -r ${fault_name} -c $comp -t o -e -s
```
default should still be geom BTW, just using 3 comps to show how it works.